deep_qa/data/instances/tuple_inference_instance.py

```diff
@@ -137,11 +137,12 @@ def __init__(self, answer_tuples: List[List[TextTuple]], background_tuples: List
 
     def display_string(self):
         to_return = 'MultipleTrueFalseTuplesWithBackgroundInstance: \n'
-        to_return += "  Answer Candidates: \n"
+        to_return += "Answer Candidates: \n"
         for answer_index in range(len(self.answer_tuples)):
-            string_answer_tuples = [str(a_tuple) for a_tuple in self.answer_tuples[answer_index]]
-            to_return += "\t [{0}] {1}\n".format(answer_index, ", \n".join(string_answer_tuples))
-        to_return += "\n  Background Tuples: \n " + ", \n".join(str(self.background_tuples))
+            string_answer_tuples = [a_tuple.display_string() for a_tuple in self.answer_tuples[answer_index]]
+            to_return += "Answer Option [{0}]:\n{1}\n".format(answer_index, ",\n".join(string_answer_tuples))
+        to_return += "\nBackground Tuples:\n" + \
+                     ",\n".join([background.display_string() for background in self.background_tuples])
         return to_return
```

Comment:
my version -- again, incorporate at will:
(NOTE: I didn't format this in the right PR way -- like, I have abbreviations for variables, etc), just thought I'd throw it out there in case it helps.  Also, I am truncating some of the data...)
```
    def display_string(self):
        to_return = "-----------------------------------------------\n"
        to_return += 'MultipleTrueFalseTuplesWithBackgroundInstance: \n'
        to_return += "  Answer Candidates: \n"
        for answer_index in range(len(self.answer_tuples)):
            string_answer_tuples = [a_tuple.display_string() for a_tuple in self.answer_tuples[answer_index][0:10]]
            if answer_index == self.label:
                to_return += "\t**"
            else:
                to_return += "\t"
            to_return += " [{0}] {1}\n".format(answer_index, ", ".join(string_answer_tuples))
        to_return += "  Background Tuples: \n " + ", ".join([bgt.display_string() for bgt in self.background_tuples[0:30]])
        return to_return
```